README.Rmd

```diff
@@ -16,104 +14,91 @@ knitr::opts_chunk$set(
 # OSPSuite-R
 
 <!-- badges: start -->
-[![](https://img.shields.io/github/actions/workflow/status/Open-Systems-Pharmacology/OSPSuite-R/main-workflow.yaml?branch=main&label=Build
-)](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/actions/workflows/main-workflow.yaml)
-[![Codecov test coverage](https://codecov.io/gh/Open-Systems-Pharmacology/OSPSuite-R/branch/main/graph/badge.svg)](https://app.codecov.io/gh/Open-Systems-Pharmacology/OSPSuite-R?branch=main)
+
+[![](https://img.shields.io/github/actions/workflow/status/Open-Systems-Pharmacology/OSPSuite-R/main-workflow.yaml?branch=main&label=Build)](https://github.com/Open-Systems-Pharmacology/OSPSuite-R/actions/workflows/main-workflow.yaml)
+[![Codecov test
+coverage](https://codecov.io/gh/Open-Systems-Pharmacology/OSPSuite-R/branch/main/graph/badge.svg)](https://app.codecov.io/gh/Open-Systems-Pharmacology/OSPSuite-R?branch=main)
+
 <!-- badges: end -->
 
 # Overview
 
-The **ospsuite-R** package provides the functionality of loading,
-manipulating, and simulating the simulations created in the Open Systems
-Pharmacology Software tools PK-Sim and MoBi.
+The **ospsuite-R** package provides the functionality of loading, manipulating,
+and simulating the simulations created in the Open Systems Pharmacology Software
```

Comment:
would replace _and simulating the simulations_ with _and **running** the simulations_